digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskLabels.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div @click.stop>
+    <v-combobox
+      ref="select"
+      v-model="model"
+      :filter="filter"
+      :hide-no-data="!search"
+      :items="items"
+      :search-input.sync="search"
+      :label="$t('homepage.task.drawer.labels')"
+      class="pt-0"
+      hide-selected
+      multiple
+      small-chips
+      prepend-icon
+      solo>
+      <template v-slot:prepend>
+        <i class="uiIconTag uiIconBlue mr-1"></i>
+      </template>
+      <template v-slot:no-data>
+        <v-list-item>
+          <span class="subheading">{{ $t('homepage.task.drawer.create') }}</span>
+          <v-chip
+            label
+            small>
+            {{ search }}
+          </v-chip>
+        </v-list-item>
+      </template>
+      <template v-slot:selection="{ attrs, item, parent, selected }">
+        <v-chip
+          v-if="item === Object(item)"
+          v-bind="attrs"
+          :color="`${item.color} lighten-3`"
+          :input-value="selected"
+          label
+          small>
+          <span class="pr-2">
+            {{ item.text }}
+          </span>
+          <v-icon
+            small
+            @click="parent.selectItem(item);deleteTaskLabel(item)">close</v-icon>
```

Comment:
Internationalization
```suggestion
            @click="parent.selectItem(item);removeTaskFromLabel(item)">close</v-icon>
```